src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -17,101 +17,12 @@
-//     * - the {@code actualModel} matches {@code expectedModel}
-//     */
-//    public static void assertCommandSuccess(Command command, Model actualModel, CommandResult expectedCommandResult,
-//            Model expectedModel) {
-//        try {
-//            CommandResult result = command.execute(actualModel);
-//            assertEquals(expectedCommandResult, result);
-//            assertEquals(expectedModel, actualModel);
-//        } catch (CommandException ce) {
-//            throw new AssertionError("Execution of command should not fail.", ce);
-//        }
-//    }
-//
-//    /**
-//     * Convenience wrapper to {@link #assertCommandSuccess(Command, Model, CommandResult, Model)}
-//     * that takes a string {@code expectedMessage}.
-//     */
-//    public static void assertCommandSuccess(Command command, Model actualModel, String expectedMessage,
-//            Model expectedModel) {
-//        CommandResult expectedCommandResult = new CommandResult(expectedMessage);
-//        assertCommandSuccess(command, actualModel, expectedCommandResult, expectedModel);
-//    }
-//
-//    /**
-//     * Executes the given {@code command}, confirms that <br>
-//     * - a {@code CommandException} is thrown <br>
-//     * - the CommandException message matches {@code expectedMessage} <br>
-//     * - the address book, filtered person list and selected person in {@code actualModel} remain unchanged
-//     */
-//    public static void assertCommandFailure(Command command, Model actualModel, String expectedMessage) {
-//        // we are unable to defensively copy the model for comparison later, so we can
-//        // only do so by copying its components.
-//        AddressBook expectedAddressBook = new AddressBook(actualModel.getAddressBook());
-//        List<Person> expectedFilteredList = new ArrayList<>(actualModel.getFilteredPersonList());
-//
-//        assertThrows(CommandException.class, expectedMessage, () -> command.execute(actualModel));
-//        assertEquals(expectedAddressBook, actualModel.getAddressBook());
-//        assertEquals(expectedFilteredList, actualModel.getFilteredPersonList());
-//    }
-//    /**
-//     * Updates {@code model}'s filtered list to show only the person at the given {@code targetIndex} in the
-//     * {@code model}'s address book.
-//     */
-//    public static void showPersonAtIndex(Model model, Index targetIndex) {
-//        assertTrue(targetIndex.getZeroBased() < model.getFilteredPersonList().size());
-//
-//        Person person = model.getFilteredPersonList().get(targetIndex.getZeroBased());
-//        final String[] splitName = person.getName().fullName.split("\\s+");
-//        model.updateFilteredPersonList(new NameContainsKeywordsPredicate(Arrays.asList(splitName[0])));
-//
-//        assertEquals(1, model.getFilteredPersonList().size());
-//    }
-//
-//}
+@SuppressWarnings("checkstyle:CommentsIndentation")
```

Comment:
Shouldn't you uncomment the JavaDoc comment above instead of adding @SuppressWarnings?
Also, line 2, 3, 4 is it supposed to be still commented?